examples/siro_sandbox/sandbox_app.py

```diff
@@ -525,7 +608,15 @@ def parse_debug_third_person(args, framebuffer_size):
             )
             task_config.actions.arm_action.arm_controller = "ArmEEAction"
 
-    framebuffer_size = gui_app_wrapper.get_framebuffer_size()
+    assert (
```

Comment:
Alternatively to this assert, after the Sandbox config is we may list the config agents and ask the user to pick which one he wants to control if any.